components/textbook-beta/CoursePagesSection.vue

```diff
@@ -83,6 +83,18 @@ export default class CoursePagesSection extends Vue {
       segment: { cta: 'what-is-quantum', location: 'course' },
       url: 'https://learn.qiskit.org/course/introduction/what-is-quantum'
     },
+    {
+      image: 'describing-quantum-computers-preview.png',
+      label: 'Describing quantum computers',
+      segment: { cta: 'describing-quantum-computers', location: 'course' },
+      url: 'https://learn.qiskit.org/course/introduction/describing-quantum-computers'
+    },
+    {
+      image: 'entangled-states-preview.png',
+      label: 'Entangled states',
+      segment: { cta: 'entangled-states', location: 'course' },
+      url: 'https://learn.qiskit.org/course/introduction/entangled-states'
+    },
     {
       image: 'multiple-qubits-and-entangled-states-preview.png',
       label: 'Multiple qubits and entangled states',
```

Comment:
we should remove this (`Multiple qubits and entangled states`) section as it is replaced by the `Entangled states` section